core/src/main/scala/kafka/server/KafkaRequestHandler.scala

```diff
@@ -89,7 +90,8 @@ class KafkaRequestHandler(
   id: Int,
   brokerId: Int,
   val aggregateIdleMeter: Meter,
-  val totalHandlerThreads: AtomicInteger,
+  val perPoolIdleMeter: Meter,
+  val poolHandlerThreads: AtomicInteger,
```

Comment:
Let's use similar naming conventions. E.g. `poolIdleMeter` and `poolHandlerThreads`.